test/functionalTest/cases/4235_wrong_response_code_not_found_reg/wrong_response_code_not_found_reg.test

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Not found registration returning 404
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0 IPV4
+
+--SHELL--
+
+#
+# 01. GET /v2/registrations/testtesttesttesttesttest get 404 Not Found HTTP code
+# 02. DELETE /v2/registrations/testtesttesttesttesttest get 404 Not Found HTTP code
```

Comment:
Not actually within scope of issue https://github.com/telefonicaid/fiware-orion/issues/4235 (actually, DELETE is working correctly when registry is not found) but we take the opportunity of also covering this case, which is pretty similar.

NTC

Reply:
LGTM